casts/cards/src/UIMediaCard/UIMediaCard.tsx

```diff
@@ -11,29 +11,54 @@ import {
 } from '@tonlabs/uikit.themes';
 import { TouchableOpacity } from '@tonlabs/uikit.controls';
 import { UILayoutConstant, UISkeleton } from '@tonlabs/uikit.layout';
+import { uiLocalized } from '@tonlabs/localization';
 import type { UIMediaCardProps } from './types';
 import { UIConstant } from '../constants';
 import { CollectionSlide } from '../UICollectionCard/CollectionSlide';
 
 export function UIMediaCard({ content, title, onPress, loading, testID }: UIMediaCardProps) {
     const theme = useTheme();
     const styles = useStyles(theme);
+    const [isContentLoaded, setIsContentLoaded] = React.useState<boolean>(false);
+    const [isError, setIsError] = React.useState<boolean>(false);
+
+    const onLoad = React.useCallback(function onLoad() {
+        setIsContentLoaded(true);
+    }, []);
+    const onError = React.useCallback(function onError() {
+        setIsError(true);
+    }, []);
+
     return (
         <UISkeleton show={!!loading} style={styles.skeleton}>
             <TouchableOpacity testID={testID} onPress={onPress} style={styles.touchable}>
                 <View style={styles.container}>
                     {content ? (
-                        <CollectionSlide style={styles.quickView} content={content} />
+                        <CollectionSlide
+                            style={styles.quickView}
+                            content={content}
+                            isVisible={isContentLoaded && !isError}
+                            onLoad={onLoad}
+                            onError={onError}
+                        />
                     ) : null}
                     <LinearGradient
                         start={{ x: 0, y: 1 }}
                         end={{ x: 0, y: 0 }}
                         colors={[UIConstant.linearGradientStart, UIConstant.linearGradientEnd]}
                         style={styles.gradient}
                     />
+                    <UILabel
+                        style={styles.description}
+                        role={TypographyVariants.NarrowParagraphFootnote}
+                        color={ColorVariants.TextSecondary}
+                        numberOfLines={UIConstant.uiCollectionCard.numberOfLinesInTitle}
+                    >
+                        {!content || isError ? uiLocalized.TnftNotSupportedMedia : ''}
```

Comment:
Yep, IMHO it's better to provide a more generic error message in UIKit (or use the `notSupported` message passed via props if available)